### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by wangqingl** [wgcban/ChangeFormer#89]
Hello, when I modified two categories into ten categories, the loss function dropped abnormally, and in the second round it dropped to 0. I tested the model, but it didn't work. How should I solve this problem? Looking forward to your reply
![image](https://github.com/wgcban/ChangeFormer/assets/102707449/994c9d75-dcd6-4da8-b98a-2e2c9d367582)

**Comment by wangqingl** [comment]
> @wangqingl I think something is going wrong with your setup. This loss values are very low and I do not think the model is learning anything. Please check if the labels used to compute CE loss is properly setup.

ok, thank you for your reply, I delete the pretrained model and modify the learning rate, the loss is normal

**Comment by wgcban** [comment]
Great!

**Comment by wangqingl** [comment]
> Great!

Sorry to bother you again, there was an issue with the data display during the training period. When I tested the trained model, the model also had no effect. This is trained using a multi class dataset
![image](https://github.com/wgcban/ChangeFormer/assets/102707449/2735430e-5038-48ae-b4e4-f2b29e95f458)
